internal/validators/namespace_test.go

```diff
@@ -335,3 +335,65 @@ func TestIllegalIncludedNamespaceNamespace(t *testing.T) {
 		})
 	}
 }
+
+func TestNsTreeLabelAddition(t *testing.T) {
```

Comment:
Just `TestTreeLabelAddition` is probably fine here since we're already testing namespace validators so the "Ns" is redundant.